api/inc/vmpu_exports.h

```diff
@@ -147,14 +147,42 @@ typedef struct {
 typedef struct {
     uint32_t magic;
     uint32_t version;
+
+    /* contains all volatile memory of the box */
+    uint32_t bss_size;
```

Comment:
Everything is exact, but bss_size contains the sum of it plus the rounding buffer.
I'll add a comment, this is quite important, yes.